pkgs/tools/archivers/wiseunpacker/default.nix

```diff
@@ -0,0 +1,40 @@
+{ fetchFromGitHub
+, buildDotnetModule
+, dotnetCorePackages
+, lib
+}:
+let
+  version = "1.3.3";
+  pname = "WiseUnpacker";
+in
+buildDotnetModule rec {
+  inherit version pname;
+
+  src = fetchFromGitHub {
+    owner = "mnadareski";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-APbfo2D/p733AwNNByu5MvC9LA8WW4mAzq6t2w/YNrs=";
+  };
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+
+  dotnetInstallFlags = [ "-f net8.0" ];
+
+  # Rename to something sensible
+  postFixup = ''
+    mv "$out/bin/Test" "$out/bin/WiseUnpacker"
+  '';
+
+  nugetDeps = ./deps.nix;
+
+  projectFile = "Test/Test.csproj";
```

Comment:
Should this be using the main project file, and specifying Test.csproj as `testProjectFile?

Reply:
WiseUnpacker is technically a C# library, and comes with a test project that's just a CLI tool which exposes the basic functionality of extracting files out of a Wise packed installer. I don't think it would make sense to use the main project file since it's just a library with no executable.